src/components/surface/CdrSurface.vue

```diff
@@ -9,28 +9,72 @@ import { getSurfaceProps } from '../../utils/surface';
 defineOptions({ name: 'CdrSurface' });
 
 const props = withDefaults(defineProps<surface>(), {
+    /**
```

Comment:
FYI this type of documentation is also part of the `surface` interface type. I'm not sure which one the doc site will pick up. I could see it as a little confusing to have both for someone coming into it without background knowledge, but I don't mind.

Reply:
Oh my bad - interface seems like the correct location for this - the doc site seems to be using slightly different language